_samples/json/identify_value_key_pairs.sql

```diff
@@ -0,0 +1,5 @@
+-- this will provide you with every key value pair, which will then be leverage in the stg_model
```

Comment:
suggestion: you could add the snowflake documentation https://docs.snowflake.com/en/sql-reference/functions/flatten